tasks/main.yml

```diff
@@ -138,16 +200,15 @@
 
 - name: PKCS#12 certs
   include_tasks: pkcs12_cert.yml
-  when: ((eduroam.radsec is defined) and (eduroam.radsec.certificate is undefined)) and
-        ((eduroam.EAP is defined) and (eduroam.EAP.certificate is undefined))
+  when: eduroam.radsec is undefined or eduroam.EAP is undefined
```

Comment:
Interesting condition, it took me quite a time to understand it. Ive created issue from it https://github.com/CESNET/ansible-freeradius/issues/17